src/lib.rs

```diff
@@ -43,6 +43,40 @@ mod test_utils;
 #[cfg(test)]
 mod tests;
 
+#[cfg(target_arch = "wasm32")]
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
+
+#[cfg(target_arch = "wasm32")]
```

Comment:
Why did it move up top of `contract` feature?

Reply:
It's needed for a test contract which uses the production contract as a dependency, but I can't have the production public exports (i.e. `contract` feature) because I want to override the definition in the test contract and you can't export a two methods with the same name.